Python/PRP/Surface/pyMipmap.cpp

```diff
@@ -216,21 +231,19 @@ PY_METHOD_NOARGS(Mipmap, extractColorData,
     "Extract an RGB color buffer from a JPEG mipmap")
 {
     size_t dataSize = self->fThis->getWidth() * self->fThis->getHeight() * 3;
-    char* data = new char[dataSize];
+    PyObject* buf = PyBytes_FromStringAndSize(NULL, dataSize);
+    char* data = PyBytes_AS_STRING(buf);
```

Comment:
This either needs to be added to PyPlasma.h for Python 2.x, or we need to formally drop support for Python 2.6 (I'd prefer the latter, but it's a slightly larger and more unrelated change)

Reply:
Fixed